MyCryptoHQ/MyCrypto
# TX History Improvements

[ch5999]
### Description
General improvements to TX history panel

Note: I tested `useTxMulti` and the tx worked but it didnt add to tx history. im going to go ahead and merge this so that this pr doesn't get larger than it already is, and i've m.ade a card to address tx history + `useTxMulti` integration

###### Changes
- standardizes ITxReceipt types.
- added handling based on txType.
- icon updates.
- max-height for panels.
- copy.

### Steps to Test:
Test every transaction flow with web3 and non-web3 tx. Ensure that tx confirm panels seem okay and that tx receipt panel content seems okay. Make sure txs appear correctly in tx history panel.
- [ ] DeFiZap
- - [ ] Web3/WalletConnect
- - [ ] Non-Web3
- [x] SendAssets
- - [x] Web3/WalletConnect ETH
- - [x] Web3/WalletConnect ERC20
- - [x] Non-Web3 ETH
- - [x] Non-Web3 ERC20
- [ ] Swap
- - [ ] Web3/WalletConnect
- - [ ] Non-Web3
- [ ] Membership Purchase
- - [ ] Web3/WalletConnect
- - [ ] Non-Web3
- [x] Contract Interact
- - [x] Web3/WalletConnect
- - [x] Non-Web3

![image](https://user-images.githubusercontent.com/29407814/82389521-44e13780-99f1-11ea-96ae-d65ace78e902.png)
